linelist_investigations/report_sources/age_sex_2019-10-18.Rmd

```diff
@@ -397,14 +419,17 @@ have been generated and are in the same folder as this document):
 Explain what the exported files are using the following model:

```

Comment:
Remove this line as it was part of the template.

Reply:
Deleted " (only works if the files above have been generated and are in the same folder as this document)"